### PR TITLE
fix(client): fix prisma/prisma-client-js#585

### LINKED ISSUE #585: [Javascript] Better error message than `TypeError: Cannot read property 'findMany' of undefined` for non existing model?
When you try to run a query on a model (and thus its methods) that does not exist, you get this error message:

```
C:\Users\Jan\Documents\throwaway\prisma2-client-502>node index.js
(node:11760) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'findMany' of undefined
    at main (C:\Users\Jan\Documents\throwaway\prisma2-client-502\index.js:6:46)
    at Object.<anonymous> (C:\Users\Jan\Documents\throwaway\prisma2-client-502\index.js:16:1)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:839:10)
    at internal/main/run_main_module.js:17:11
(node:11760) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:11760) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Could this possibly be handled in some way and give the user a more helpful error message? 

(In my case I messed up the spelling of a model in a quite obvious way that some software could probably suggest the correct name [organization => orgnization]. Same might be possible for lowercasing problems)

### PR DESCRIPTION
**Pull request by timsuchanek**
Closes prisma/prisma-client-js#585

### CODE REVIEW

File: src/packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts [hunk 1]
```diff
@@ -51,10 +51,6 @@ describe('blog-env-mssql', () => {
     expect(prismaVersion.client).not.toBeUndefined()
   })
 
-  test('does not leak connection strings in node_modules', () => {
-    expect(prisma.internalDatasources).toBeUndefined()
```
Comment (Jolg42) [inline src/packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts:55]: Why removing this? Is this done in another test now or not relevant anymore?

File: src/packages/client/src/__tests__/integration/happy/blog-env-postgresql/test.ts [hunk 2]
```diff
@@ -53,10 +53,6 @@ test('Blog fixture: Postgres', async () => {
     ],
   })
 
-  // Make sure we're not leaking connection strings in node_modules
-
-  expect(prisma.internalDatasources).toBe(undefined)
```
Comment (Jolg42) [inline src/packages/client/src/__tests__/integration/happy/blog-env-postgresql/test.ts:58]: Why removing this? Is this done in another test now or not relevant anymore?

File: src/packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts (hunk 1 above)
Comment (timsuchanek) [reply src/packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts:55]: We don't have them anymore.

File: src/packages/client/src/__tests__/integration/happy/blog-env-postgresql/test.ts (hunk 2 above)
Comment (timsuchanek) [reply src/packages/client/src/__tests__/integration/happy/blog-env-postgresql/test.ts:58]: We don't have them anymore.

**Pull request merged**